sys/include/net/gnrc/netif.h

```diff
@@ -41,6 +41,11 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Identifier of the loopback interface
+ */
+#define GNRC_NETIF_LOOPBACK_PID     (-1)
```

Comment:
I'd rather have it a positive number (maybe `KERNEL_PID_MAX`?), since I think there might be checks for `>= KERNEL_PID_UNDEF` in the code.